content/200-concepts/150-data-platform/02-data-proxy.mdx

```diff
@@ -95,7 +95,15 @@ To generate Prisma Client with the Data Proxy, call `prisma generate` with the o
 npx prisma generate --data-proxy
 ```
 
-When you use `--data-proxy`, you get a very lightweight Prisma Client runtime of approximately 250kb. This is because the Data Proxy does not need local engine files, so the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine).
+Alternatively, you can set the environment variable `PRISMA_GENERATE_DATAPROXY=true` to generate a Prisma Client with Data Proxy when a normal `prisma generate` command is executed.
+
+<Admonition type="warning">
+
+When deploying to [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) you _need_ to define `PRISMA_GENERATE_DATAPROXY=true` [as an environment variable in your Vercel project settings](https://vercel.com/docs/concepts/projects/environment-variables) for the correct Prisma Client for Data Proxy to be generated.
+
+</Admonition>
+
+When you use `--data-proxy` or `PRISMA_CLI_GENERATE_DATAPROXY=true`, you get a very lightweight Prisma Client runtime of approximately 250kb. This is because the Data Proxy does not need local engine files, so the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine).
```

Comment:
```suggestion
When you use `--data-proxy` or `PRISMA_CLI_GENERATE_DATAPROXY=true`, you get a very lightweight Prisma Client runtime. This is because the Data Proxy does not need local engine files, so the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine).
```